tabs/index.js

```diff
@@ -0,0 +1,83 @@
+const {tabs} = require('../firebase');
+
+const DEBTS_TYPES = {
+  pizza: '🍕',
+  churras: '🥩',
+  cerva: '🍺',
+};
+
+function formattedTab(tab) {
+  let debts = '';
+  for (let type in DEBTS_TYPES) {
+    let debt = tab[type];
+    if (debt && debt > 0) {
+      if (debts.length > 0)
+        debts += ', ';
+      debts += `${DEBTS_TYPES[type]}: ${debt}`;
+    }
+  }
+  if (debts.length == 0)
+    return '';
+
+  return `${tab.debtor} = { ${debts} }\n`;
```

Comment:
What do you think?
```suggestion
  let debts = [];
  for (let type in DEBTS_TYPES) {
    let debt = tab[type];
    if (debt && debt > 0) {
      debts.push(`${DEBTS_TYPES[type]}: ${debt}`);
    }
  }
  
   if (debts.length == 0)
    return '';
  return `${tab.debtor} = { ${debts.join(', ')} }\n`;
```

Reply:
I prefer to avoid the creation of the intermediate array and the need to loop through the data twice (`join` is a loop). I don't see much gain in readability, in this case, to make up for the memory overhead.

Reply:
Yeah, it makes sense. Thanks for explanation